evennia/utils/evstring.py

```diff
@@ -0,0 +1,1231 @@
+
+# ------------------------------------------------------------
+#
+# EvString - ANSI-aware string class
```

Comment:
I suppose this is aware of evennia-style |-formatting rather than ANSI per-se, so this docstring is a bit misleading.